packages/qdrant-loader/tests/unit/core/pipeline/workers/test_embedding_worker.py

```
@@ -578,23 +584,26 @@ async def consume():
             assert len(cancelled) == 2, (
                 f"expected both in-flight batches cancelled, got {len(cancelled)}: "
                 f"{cancelled}"
             )
         finally:
             hold.set()  # unblock any leaked task so the loop can close cleanly
-            await asyncio.sleep(0)
+            consumer.cancel()
+            try:
+                await asyncio.wait_for(consumer, timeout=5)
+            except (asyncio.CancelledError, asyncio.TimeoutError):
+                pass
```

Ruff (0.15.21)

[warning] Use `contextlib.suppress(asyncio.CancelledError, asyncio.TimeoutError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(asyncio.CancelledError, asyncio.TimeoutError): ...`

(SIM105)